File: pset6/mario/more/mario.py
while True:
    height = int(input("specify the height of pyramid between 1 and 8\n"))
    if height >= 1 and height <= 8:
        break

for i in range(height):
    for j in range(2*height +2):
        if (j >= height - 1 -i and j < height) or (j <= height + 2 + i and j > height + 1):
            print("#", end = "")
        else :
            print(" ", end = "")
    print()

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

void formbricks (int n);

int main (void)
{
    int h;
    do 
    {
    h = get_int("specify the height of pyramid between 1 and 8\n");    
    }
    while (h < 1 || h > 8);
    formbricks (h);
    
}

void formbricks (int n)
{
    for(int i = 0; i < n; i++)
    {
        for(int j = 0; j < (2*n+2); j++)
        {
            if ( (j-n) < -i - 1 || (j-n)==0 || (j-n)> i+2 || (j-n)==1)
            {
                printf(" ");
            }
            else 
            {
                printf("#");
            }
        }
        printf("\n");
    }
}